.travis.yml

```diff
@@ -1,40 +1,49 @@
-language: generic
-
-env:
-  global:
-    - secure: "Zp+Zww6gqU0xNUcrOz0Xm2Uy8AflwJX3GZxydGs4YfClEIsCtK/BzVv3lzp2N4ouwRUkEsYxX6r+MUSV/ribaizCMspjTOvmaKgN/n0vg6cMTadCezxaXYLR2NDmF8ydkA/OCShJWxnRl4myZumm2nCS7rNmq43AliO7uHK43SGFTgfyzSR6ua2Imw/IGDnaZH31CuOzQlagEDuVdLuGZzulSW8b7nvalN1dHtRKYBUx3aq8fZd7VTUdvuGzsQyGoD+/VD4hjcJ7Y9R6EoCAdsesfxt2XCu6ZjtJ+cmi2bAbjJJYCtBjdkLYFguwy6XTObjwWKkGFDlMvkChkq7AnBaVNRNtyHHutkvZKTQQQ58iswOybOx3TTat5+uatl1XFgkVItuE8Yng+Cm1heZDBwYsz6690dACS4PqUGn5E9lA2RJsH4YkOAhV7HggV7bK5iPeSObRx4Pj1jDzIx+apOAzUDSrVBCaTS1bAKOeZPr/erqavp1anydkcPFdEQ0/tixkUXrmneh+IbhHkbRYfBH4xLIklPTX8Ko2prS+P9dOd0dkFYrF5iPzGPhYwvbZJ/E9LplXZ/ylKz9fxDCIpsnYvBImT75yUg6uoGEJQxOsOXOBMTEVLQ91mexBK0SoQP16yQ0QOV+kitEBv9zwFU3f8Mhr7lICl9wJlByoBRY="
-    - secure: "stmKysfUb9uDgODLB4cUdkQm++T306QxdX4Yr7Ck7T9Yf0y/VUgYRGQKT8bC0RXpdE3jvlZsh8AJnVNY2VOKGq98y2PkiZHnD60Ncbk36uMhI6LV3z65mCGjzUIlNCM0Agxb6W2CUEiVy30Ohnl78Xq8mfWK+UfHC4VQTr7uUCuaqHR3cZuTlLwgjKWBNNN/qbXbjGM7zAhl5UsqkQFcJflq8fs2QKlnuLL1rk6cDvITb+x/RnuN+jt2uTPrg9a01YJQtQebri6/MhzTz/EG3LJkjX+ikbtnqKyG5LRhnpn4jARjDAV0Aa371wXoZ9txen6+HfoT/Ii4musRNQAOsnt0Q5UksSUbWeol45DwoBhMtdvz65hedpLe3Onzw12DIga+agyhHfLTolzeIriLWccujnPG4QmaHYmDOTkXMAC97Vhw2KOzmuakEghcJWNdEzKMnt2DtYrBHY4/o7lUxArWjoeDUrAeCOZXu3NfOq1oQdA7gORO/fDWSXzcJEatA0Eswd4yYpB35ItQX87X2gAjcTSClhFsMTDatgV2nWqIrFJYd6IOpWxihsyFL8+hOZzw8clDKBNMR9wi/TVgOP1A2UyKYLvbvs721MybZ2pO883kHN0jcrLrZTMg5Awsa2Cet4aLJzwdFtIrtq+UC+9ojtMtPM6EtMzsSQWjJI4="
-    - secure: "lB+X2gQUGKSpcMgPAsL7LdL/0m68pzutLTT5fPh2JJWjAQsJ3a6cqMEiHUsRI8CX01IcuuJ7O+MsRy4/iWvI8UdZsT83sT4sGa9nihLsoldchlmAvo4K4udNpGoXveqkpDGf8TEqEj7OA8JF8L1XTAr63/fU6kzXboV1p4xGaQjqo/OdQBellvIbh4mcZnvE0l+yBRRus9vvgrQEZ6wfQ/CzE9PjMxoveVy1r8G5vhbvi3+/h8aTKI3exH7WZd5NucDk0X6Zkzir7aOnwrBjkgcj25yMoxcFZ4ZBmThBbOE5xFLPAa/3jFNbs2ZorSEims/OXPTV8pTW7WdM1Fa2gSQRiaoZu4ARZ0e+VSTFq/hVJvEMR/fm6V+nWPGzLAG+BhIIlrDgDZ9aAq1ANq3SfFNbq4W1LKVqDwQresQqg9WATHZJ9ZanU9vKwsTabeQKZm9n+BJaoQlE6KOUiNZ85gB79JrWkGdcuLDqB1EKHgABXaygT6RI0kI++z4/FFbBFdf8N8OgR03LiabE6Gp8UpMCFY1/JUBlXV+hncKt8PxhxKa2eUr8uls3J7TFa+0BM5AClM9GTbXnYWbmM0Gcy9ewnk3qSSvJak7ghusuoFM7X80OPSrQyfxPSfBGqu0E0aDSJfHOllfC6g7i8fndr2pbOlEz5HXukOTB5/xRxiM="
-    - secure: "IeGJF78ctI4YbBZDi4E6ziOFIMqhw05InvvdT5Do7BaFaKtHLMdSfEFL6IDcuD0932l6m49S2hWNHTWwkWCMp5EqZ1YX6FqpaaXOquN231h6bRA4CtT/1TCouZQ35Q7HZwdwx305pBYRs9/2MB7uf6qAjZwtkWOSuyeBEXYo1vmXizAPNrb/TyjZNmDQ2VqlEgzxUkMm3nhrvbq2y2GeRMf97qXguKnlqcRFXM2ABZUUsQQ8P4BFAF/coGzfAcptIiWbWrfz0wMPhZrVwdAy3Ul647P3npEluOXbFbGkldoFDRv1WaenJA5pk6RtE73HiOitLLf91XDFSeqeYmY5tYrwS44IC7L02tsRTtA/ZHSk6hY8kRq7/s1HZetAjfkZ4VAUBj5bQ+l/fvgh7nNMwsQ6R0EMAipeIMzrMjHmZBoFlJdO1J198RvtcPrdfxbZcHcRKDgNXxnIw9W6KNoC+OCZppQb+pRazr0k+4Bc1G9B38n+4Ntak8u0L5GGebdO5VHBln00h6b2NzEI089JOa8CkocyQV89F3AXOOwubS3oXVWoQCoXkU9/AhYSgMgQ5zH4ky7aV6y5mq5sByM4sEOuo9SFMVnc7gM6cduKyVKbhjY/WriqesGV709JqspgQcqbw6u0ShDntcKEQlz9n1d4gOGWNjo4UMrPqLrNK/w="
-
+language: python
 dist: trusty
 sudo: required
```

Comment:
As you now use `addons.apt` you should be able to get rid of `sudo: required`? That would mean much faster Travis builds.